README.md

```diff
@@ -1,2 +1,11 @@
-# oauth2-service
-OAuth2.0 Service.
+# OAuth2-service
+
+OAuth2.0 Interface service for the PlayTIX SSO.
+
+## What is it?
+
+An OAuth2.0 interface to authenticate users from the Okta's Playtix organization.
+
+## How to setup Dev environment
+
+WIP
```

Comment:
I'm sure you're still working on this, let's make sure we don't forget to add the basic setup instructions :)

Reply:
Of course yes. I planned to write it once this PR gets accepted to that the setup is based on a default version.